arkane/kinetics.py

```diff
@@ -71,34 +71,28 @@ def __init__(self, reaction,
                  sensitivity_conditions=None):
         self.usedTST = False
         if Tmin is not None:
-            self.Tmin = quantity.Quantity(Tmin)
+            self.Tmin = Tmin
         else:
-            self.Tmin = None
+            self.Tmin = (298, 'K')
 
         if Tmax is not None:
-            self.Tmax = quantity.Quantity(Tmax)
+            self.Tmax = Tmax
         else:
-            self.Tmax = None
+            self.Tmax = (2500, 'K')
 
+        if Tcount <= 3.:
+            Tcount = 50
         self.Tcount = Tcount
 
         if Tlist is not None:
-            self.Tlist = quantity.Quantity(Tlist)
-            self.Tmin = quantity.Quantity(numpy.min(self.Tlist.value_si), "K")
-            self.Tmax = quantity.Quantity(numpy.max(self.Tlist.value_si), "K")
+            self.Tlist = Tlist
```

Comment:
maybe skip this assignment, and just use Tlist?

Reply:
I would need to assign something to `self.Tlist` at some point, since it is used in `generateKinetics`.

I recognized the `self.Tlist.value_si` looks a little clunky, but since the Tlist passed to this method is of the format `([100,200,500,800,1200], 'K'), I would need to use `Tlist[0]` to get the array, which is not very clear to someone just looking at this method.